### ISSUE TITLE
parking

### ISSUE THREAD

**Issue by kainagel** [matsim-org/matsim-code-examples#1184]
1. According to https://wiki.vsp.tu-berlin.de/node/159, Gregor R., @simei94 and @paulheinr wanted to look into parking.
2. @jakobrehmann has a "parking" task in the FABILUT project.

We (incl. @kainagel) should sit together (soon) during one of these mondays to discuss how to proceed.  Afterwards, we may have to move it into one of the lower prio categories.

**Comment by paulheinr** [comment]
Yes, I am currently working on https://github.com/matsim-org/matsim-libs/issues/3523

**Comment by paulheinr** [comment]
It is implemented technically in matsim-libs, but need to find out good configurations.

**Comment by GregorRyb** [comment]
- add more comments in code
- move everything into the matsim parking contrib 
- parkingSearchQueue Name anpassen in etwas generelleres (arriving queue)
- See branch: vsp/parkingDocs
